azkaban-web-server/src/main/java/azkaban/webapp/Status.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2017 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ */
+
+package azkaban.webapp;
+
+import azkaban.executor.Executor;
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+
+/**
+ * This POJO is used by GSON library to create a status JSON object. Certain warnings do not apply
+ * here.
+ */
+@SuppressWarnings({"FieldCanBeLocal", "unused"})
+public class Status {
```

Comment:
The reason I kept it `Status` is because the code is already in the web server module in the `azkaban.webapp` package. And this class is intended to provide the status of both the web and executor. Future changes will pull in the executor status into the web server.

So, it is kinda aimed at being the status of the entire system.